store/sqlstore/upgrade.go

```diff
@@ -707,9 +707,8 @@ func UpgradeDatabaseToVersion513(sqlStore SqlStore) {
 }
 
 func UpgradeDatabaseToVersion514(sqlStore SqlStore) {
-	// TODO: Uncomment following condition when version 5.14.0 is released
-	// if shouldPerformUpgrade(sqlStore, VERSION_5_13_0, VERSION_5_14_0) {
-
-	// 	saveSchemaVersion(sqlStore, VERSION_5_14_0)
-	// }
+	if shouldPerformUpgrade(sqlStore, VERSION_5_13_0, VERSION_5_14_0) {
```

Comment:
You have to add your migrations in between the comments, only when the release is generated we remove the comments.